src/new-client/vite.config.ts

```diff
@@ -124,6 +124,12 @@ const setConfig = ({ mode }) => {
     },
     server: {
       port: 1917,
+      proxy: !process.env.VITE_MOCKS && {
+        "/ws": {
+          target: "http://localhost:55000",
```

Comment:
we need this locally, as we have to run through the nginx running in Docker with its associated config - going direct to wss:<demo WS url> does not work as I think we need those special proxy args - we may be able to apply them to the local vite (http-proxy) config - any thoughts, @garyadaptive ?